src/promptflow-tracing/promptflow/tracing/_openai_utils.py

```diff
@@ -211,9 +211,14 @@ def __init__(self, response, is_chat):
 
     @property
     def model(self):
-        for item in self._response:
-            if hasattr(item, "model"):
-                return item.model
+        """
+        This method iterates over each item in the _response list.
+        If the item has a non-empty 'model' attribute, it returns the model.
+        If no such item is found, it returns None.
+        """
+        for response_item in self._response:
+            if hasattr(response_item, "model") and bool(response_item.model):
```

Comment:
Why `bool()` here?

Reply:
There is empty model in response, which is `''`. This will lead model of streaming mode to be empty in UI.

Reply:
How about use the str itself to check, not using bool?
```suggestion
            if hasattr(response_item, "model") and response_item.model:
```